src/main/java/com/sparrowwallet/sparrow/io/CardApi.java

```diff
@@ -45,6 +47,14 @@ public static List<WalletModel> getConnectedCards() throws CardException {
             //ignore
         }
 
+        try {
+            log.error("SATOCHIP in CardApi getConnectedCards() new SatoCardApi()");
+            SatoCardApi satoCardApi = new SatoCardApi(null, null);
```

Comment:
instead of creating a new SatoCardApi instance, why don't use directly the `WalletModel.SATOCHIP`?